keps/sig-release/1731-publishing-packages/kep.yaml

```diff
@@ -19,7 +20,7 @@ approvers:
   - "@tpepper"
 editor: TBD
 creation-date: 2019-02-19
-last-updated: 2022-10-12
+last-updated: 2023-01-24
 status: provisional
```

Comment:
this needs to be implementable

Reply:
I think we also need to add the milestone block in here.

Reply:
Both are done.